docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2.2'
 
 services:
   initializr:
-    image: hub.xenit.eu/start.xenit.eu
+    image: ${DOCKER_IMAGE}
```

Comment:
Can you change that into `${DOCKER_IMAGE:-hub.xenit.eu/start.xenit.eu}` ? We're actually using that compose-file to host the actual stie.